Makefile

```diff
@@ -94,6 +93,9 @@ verify-vendor-licenses:
 .PHONY: verify-crds
 verify-crds:
 	hack/verify-crds.sh
+.PHONY: verify-apidocs
+verify-apidocs:
+	hack/verify-apidoc.sh
 
```

Comment:
Maybe not deleting the newline on line 84 and adding a newline before line 96 can solve the CI error